### ISSUE TITLE
test suite fails locally

### ISSUE THREAD

**Issue by rsanheim** [reborg/pomodori#2]
Test suite fails locally on my macbook pro - looks like a time zone thing.  I'm in EDT, btw.

```
    ~/src/oss/pomodori (master)$ macrake
    (in /Users/rsanheim/src/oss/pomodori)
    Loaded suite test/all_tests
    Started
    ...................................................................................F..
    Finished in 3.465789 seconds.

      1) Failure:
    test_format the time using same Time convention(TimestampTest) [/Users/rsanheim/src/oss/pomodori/vendor/mhennemeyer-matchy-0.3.3/lib/matchy/built_in/operator_expectations.rb:20]:
    Expected "07:06pm" to == "06:06pm".

    86 tests, 147 assertions, 1 failures, 0 errors, 0 skips
```

**Comment by reborg** [comment]
Yes, thank you for reporting, someone already told me I have this unwanted time dependency. Will fix soon.

**Comment by reborg** [comment]
Fixed in 0.3
